CHANGES.md

```diff
@@ -46,12 +46,30 @@
   of the typed hole, i.e., `_|`, where `|` is the cursor. The code action simply triggers
   the client (currently only VS Code is supported) to show completion suggestions. (#472)
 
+- Add support to turn on/off using local context in `Construct an expression`. If turned
+  on, the constructed expressions list will include locally defined values.
+  (Currently, works only with VS Code OCaml Platform extension) (#481)
+
+  Example:
+
+  ```ocaml
+  type t = { name: string }
+
+  let john : t =
+    let doe = { name = "John" } in
+    _ (* <- construct is triggered on this typed hole *)
+  ```
+
+  With `Local` mode _on_, the suggested values will include the local variable `doe`
+  along with other expressions such as `{ name = _ }`.
```

Comment:
What would be the advantage of turning off this "Local" mode? To me it seems like a straight up improvement.